LEGO1/realtime/vector.h

```diff
@@ -301,10 +302,14 @@ class Vector4 : public Vector3 {
 	// FUNCTION: BETA10 0x10048780
 	Vector4(float* p_data) : Vector3(p_data) {}
 
-	// Hack: Some code initializes a Vector4 from a (most likely) const float* source.
-	// Example: LegoCarBuild::VTable0x6c
+	// Some code initializes a Vector4 from a `const float*` source.
+	// Example: `LegoCarBuild::VTable0x6c`
 	// Vector4 however is a class that can mutate its underlying source, making
 	// initialization with a const source fundamentally incompatible.
+	// BETA10 appears to have two separate constructors for Vector4 as well,
+	// supporting the theory that this decompilation is correct.
+
+	// FUNCTION: BETA10 0x100701b0
```

Comment:
I found another Vector4::Vector4 implementation in BETA10, increasing the likelihood that the two constructors is indeed what the original code used.